weatherman.py

```diff
@@ -0,0 +1,41 @@
+"""
+Task Details: https://mixolydian-bugle-21c.notion.site/Weather-Report-83c9f514a3974e5986aea2568f6f0c15
+"""
+
```

Comment:
import should be at the top of the file. Doc string add with functions and classes.